added clarity booking info to NHS admin

Lucrezia-Cester opened the pull request:
I have added the clarity booking info under the NHS Admin section of the handbook as per https://github.com/GSTT-CSC/gstt-csc.github.io/issues/134. Let me know if this is where you would expect this info to be stored at.

Lucrezia-Cester committed:
added clarity booking info to NHS admin

hshuaib90 reviewed (changes requested):
Great thanks for doing this @Lucrezia-Cester ! Can we add some text that the first step is to get travel authorised by emailing Haris. Also instead of "the site url" for the hyperlink, can you jsut add it to "Clarity Booking System"

Lucrezia-Cester committed:
action on comments

Changed the link word from URL to Clarity Booking System and added Haris's authorisation step.

Lucrezia-Cester commented:
ok, done! @hshuaib90
